recipes/libsigcpp/2.x.x/test_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake"
```

Comment:
```suggestion
    generators = "cmake", "cmake_find_package_multi"
```
It generates libsigcpp-config.cmake

Reply:
since i will be using pkg-config, I assume I won't need "cmake_find_package_multi"

Reply:
Correct! 👍  But you will need to add the pkg_config generator: https://docs.conan.io/en/latest/reference/generators/pkg_config.html